warehouse/accounts/security_policy.py

```diff
@@ -214,6 +214,8 @@ def _check_for_mfa(request, context) -> WarehouseDenied | None:
         "manage.account.totp-provision",
         "manage.account.two-factor",
         "manage.account.webauthn-provision",
+        "manage.unverified-account",
+        "account.verify-email",
```

Comment:
It's `accounts.verify-email`, and it shouldn't be necessary in this scope, since `_check_for_mfa` only happens once the user has a verified email.

Now that I think of it, neither of these should be necessary here

Reply:
Aha, good point, I missed that that was guarded.

Reply:
Odd that when removing them I get an infinite redirect, so leaving them in place sohuld be fine for now.